website/content/docs/connect/config-entries/service-intentions.mdx

```diff
@@ -18,10 +18,10 @@ The following outline shows how to format the service intentions configuration e
 <Tab heading="HCL and JSON" group="hcl">
```

Comment:
This file is best viewed w/ whitespace disabled, since I think some commit hooks are fixing trailing whitespace despite me trying to keep the diff tight